### ISSUE TITLE
Reliables visual studio code static download links (guaranteed to never break).

### ISSUE THREAD

**Issue by Luxcium** [microsoft/vscode-docs#4591]
#  tl;dr

**I feel like it should be easier to have access to a list of reliables static download links for vscode (guaranteed to never break).**

 - Where can I find reliables visual studio code permalink for each download channel or an explanation on how to construct such link? 
 - I refer to the vscode docs [faq/previous-release-versions](https://code.visualstudio.com/docs/supporting/faq#_previous-release-versions) but my goal is to get the latest version so it is not intuitive to follow the instructions to « download the *latest* previous-release-version  » 
[![Screenshot_20210605_094931](https://user-images.githubusercontent.com/42672814/120893887-7bc9c180-c5e3-11eb-8984-f49e2712cb8d.png)](https://code.visualstudio.com/docs/supporting/faq#_previous-release-versions)

  - Is the macOS dowload link include <kbd>Universal</kbd>  <kbd>Intel Chip</kbd>  <kbd>Apple Silicon</kbd> in a same .zip file?
 

## Detailed situation

I want to automate the download process of VS code (i.e. using curl or wget) the information I found in different gh issue, subreddits, StackOverflow, etc... is never consistent or is listing only one os or version or download channel  (stable vs. insiders or Debian vs Fedora, etc.) 

 - [![Screenshot_20210605_094852](https://user-images.githubusercontent.com/42672814/120893919-addb2380-c5e3-11eb-8ccd-d583a1d3d787.png)](https://code.visualstudio.com/docs/supporting/faq#_previous-release-versions)

The only information I found to be valid is in the visual studio code docs [faq/previous-release-versions](https://code.visualstudio.com/docs/supporting/faq#_previous-release-versions)... Since I am doing my project on Fedora and using only **latest/linux-x64/stable** I think I am covered... but each time I need the information I do not know where to find it (maybe after writing a gh issue it will stick in my memory)...    

 - [![Screenshot_20210605_094921](https://user-images.githubusercontent.com/42672814/120893906-9b60ea00-c5e3-11eb-8256-0f751ee5864a.png)](https://code.visualstudio.com/docs/supporting/faq#_previous-release-versions)

I was previously using download links that are in a different format:

![Screenshot_20210605_100127](https://user-images.githubusercontent.com/42672814/120894184-2a223680-c5e5-11eb-846b-fee04fa512fa.png)

```bash
      VSCODEURL_STABLE_RPM='https://go.microsoft.com/fwlink/?LinkID=760867'
      VSCODEURL_STABLE_TAR_GZ='https://go.microsoft.com/fwlink/?LinkID=620884'
      VSCODEURL_INSIDERS_RPM='https://go.microsoft.com/fwlink/?LinkID=760866'
      VSCODEURL_INSIDERS_TAR_GZ='https://go.microsoft.com/fwlink/?LinkId=723968'
```

I found partial information in [#1084 ― permalink to latest VSCode distributable](https://github.com/microsoft/vscode/issues/1084#issuecomment-162704093) and in  [#1019 ― Can't download for Linux x64](https://github.com/microsoft/vscode/issues/1019#issuecomment-162703681)

right-clicking in the download page to copy any links gives back `https://code.visualstudio.com/download#` 
 - ![Screenshot_20210605_102256](https://user-images.githubusercontent.com/42672814/120894768-26dc7a00-c5e8-11eb-9e1d-e057700ed07d.png)

**Comment by gregvanl** [comment]
Added link to /download page to make it easier to find the FAQ section.

![image](https://user-images.githubusercontent.com/12818376/129642193-0199b00a-369d-4bc3-afc4-38db84258e0c.png)
